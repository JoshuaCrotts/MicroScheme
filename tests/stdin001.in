(display "Enter two numbers.")
(define num1 0)
(define num2 0)
(set! num1 (read-number))
(set! num2 (read-number))
(display "Their sum is")
(display (+ num1 num2))

(display "Enter two more numbers.")
(set! num1 (read-number))
(set! num2 (read-number))
(display "Those two sum to")
(display (+ num1 num2))
