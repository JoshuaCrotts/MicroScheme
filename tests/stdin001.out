; Output depends on the user's input, but it should read two
; numbers, display their sum, read in two more integers (which
; overwrites the previous variable's values, then displays the
; new sum.
